modules/agent-installer-architectures.adoc

```diff
@@ -27,7 +27,7 @@ $ ./openshift-install version
 .Example output
 [source,terminal]
 ----
-./openshift-install 4.18.0
+./openshift-install 4.19.0
```

Comment:
Honestly this is probably something that could be changed to the product version attribute to save you from a bit of future work:
```suggestion
./openshift-install {product-version}.0
```